wordpress-mobile/gutenberg-mobile
# Fix iOS bridge method to gather the content html

This PR fixes the iOS side of the bridge method which gathers the content html.

### Test

- Checkout the branch
- Do usual yarn install & yarn start:reset
- Run example app gutenberg.xcodeproj via Xcode
- Tap Save button
- You shouldn't see the below crash:

<img width="380" alt="screen shot 2018-11-16 at 13 28 16" src="https://user-images.githubusercontent.com/5032900/48616052-8a0f4e00-e9a3-11e8-82b2-8b96b240c808.png">

- You should see the log "Did receive HTML: ..." in Xcode output.
